css/src/components/tag.scss

```diff
@@ -38,6 +38,9 @@ $tag-border-color-primary: $primary-background-glow-high-contrast !default;
 $tag-interactive-divider-color-primary: $primary-box-shadow !default;
 $tag-interactive-color-primary-hover: $primary-dark-hover !default;
 
+$tag-suggestion-radius: 0.5rem !default;
```

Comment:
For radius, I'd suggest just using a pixel value because we don't really want border radius's to scale based on the document's font size.